spec/lib/report_formater/timeline_spec.rb

```diff
@@ -28,4 +28,28 @@ def stub_bottleneck_event(resource_type, ems_type = nil)
       expect(ReportFormatter::ReportTimeline.new.bubble_icon(stub_bottleneck_event(ExtManagementSystem, 'rhevm'))).to eq('vendor-redhat')
     end
   end
+
+  context '#tl_href' do
```

Comment:
In general, context for a scenario/some sort of state and describe for methods is preferred, yes. Easier to visually parse faster.